torchmetrics/average.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+import torch
+
+from .metric import Metric
+
+class AverageMeter(Metric):
+    """Computes the average of a stream of values.
+
+    Forward accepts
+    - ``value`` (scalar): 
+    - ``weight`` (scalar): the weight to give this observation.
+    """
+    def __init__(self, *args, **kwargs) -> None:
```

Comment:
please be explicit about these args
```suggestion
    def __init__(self, 
        compute_on_step: bool = True,
        dist_sync_on_step: bool = False,
        process_group: Optional[Any] = None,
        dist_sync_fn: Callable = None,) -> None:
```